docs/docs/10-addons.md

```diff
@@ -18,7 +18,7 @@ The React add-ons are a collection of useful utility modules for building React
 
 The add-ons below are in the development (unminified) version of React only:
 
-- [`TestUtils`](test-utils.html), simple helpers for writing test cases (unminified build only).
-- [`Perf`](perf.html), for measuring performance and giving you hint where to optimize.
+- [`TestUtils`](test-utils.html), simple helpers for writing test cases.
+- [`Perf`](perf.html), a performance profiling tool for finding optimizations opportunities.
```

Comment:
You're totally right! I'll fix that

On Mon, Jan 25, 2016 at 2:31 AM Baraa Hamodi <notifications@github.com>
wrote:

> In docs/docs/10-addons.md
> <https://github.com/facebook/react/pull/5912#discussion_r50660214>:
>
> > @@ -18,7 +18,7 @@ The React add-ons are a collection of useful utility modules for building React
> >
> >  The add-ons below are in the development (unminified) version of React only:
> >
> > -- [`TestUtils`](test-utils.html), simple helpers for writing test cases (unminified build only).
> > -- [`Perf`](perf.html), for measuring performance and giving you hint where to optimize.
> > +- [`TestUtils`](test-utils.html), simple helpers for writing test cases.
> > +- [`Perf`](perf.html), a performance profiling tool for finding optimizations opportunities.
>
> Nit pick: optimizations should probably just be optimization here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react/pull/5912/files#r50660214>.
>